code/modules/telesci/gps.dm

```diff
@@ -50,19 +52,34 @@ var/list/GPS_list = list()
 	popup.set_title_image(user.browse_rsc_icon(src.icon, src.icon_state))
 	popup.open()
 
+/obj/item/device/gps/examine(mob/user)
+	if (Adjacent(user) || isobserver(user))
+		src.attack_self(user)
+	else
+		..()
```

Comment:
..() in all eamine procs creates "Thats a [src]! [desc] [size if item]" basically so yeah if he wants to override it this is pretty much how to do it